src/mapml/layers/FeatureIndexOverlay.js

```diff
@@ -0,0 +1,167 @@
+export var FeatureIndexOverlay = L.Layer.extend({
+    onAdd: function (map) {
+        let svgInnerHTML = `<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 100 100"><path fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="3" d="M0 0h100v100H0z" color="#000" overflow="visible"/></svg>`;
+
+        this._container = L.DomUtil.create("div", "mapml-feature-index-box", map._container);
+        this._container.innerHTML = svgInnerHTML;
+
+        this._output = L.DomUtil.create("output", "mapml-feature-index", map._container);
```

Comment:
The output element should have the following attributes:

- `role="status"`
- `aria-live="polite"`
- `aria-atomic="true"`

Motivated by https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/396:

> ```html
> <!-- 
> <output> is implicitly role="status" in supporting browsers.
> It's recommended to explicitly set the role to bridge the gaps:
> https://www.scottohara.me/blog/2019/07/10/the-output-element.html#bridging-gaps
> -->
> ```

And [MDN live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#roles_with_implicit_live_region_attributes):

> To maximize compatibility, add a redundant `aria-live="polite"` when using this role.